### PR TITLE
HTTP update server can handle flash and spiffs files (spiffs size lim…

### PR DESCRIPTION
**Pull request by PA4WD**
HTTP update server can handle flash and spiffs files (spiffs size limited by free space)

### CODE REVIEW

File: libraries/ESP8266HTTPUpdateServer/src/ESP8266HTTPUpdateServer.cpp [hunk 1]
```diff
@@ -8,12 +8,19 @@
...
+               </form>
+	       <form method='POST' action='' enctype='multipart/form-data'> 
+		  <input type='hidden' name='cmd' value='100'>
+                  <input type='file' name='update'>
+                  <input type='submit' value='Update Spiffs'>
                </form>
          </body></html>)";
 const char* ESP8266HTTPUpdateServer::_failedResponse = R"(Update Failed!)";
 const char* ESP8266HTTPUpdateServer::_successResponse = "<META http-equiv=\"refresh\" content=\"15;URL=\">Update Success! Rebooting...";
+int _command;
```
Comment (igrr) [inline libraries/ESP8266HTTPUpdateServer/src/ESP8266HTTPUpdateServer.cpp]: Could you please make the `int _command` a class member in `ESP8266HTTPUpdateServer`?

**Comment by glynhudson** [comment]
Any update on this? It would be fantastic to be able to upload and update SPIFFS via http update server. 

Thanks

**Comment by Sn4k3r** [comment]
It does not work but I fixed it!
Code should be:

action='?cmd=0'
action='?cmd=100' 

See more at: http://www.esp8266.com/viewtopic.php?f=32&t=14363#sthash.fK8iDcha.dpuf

File: libraries/ESP8266HTTPUpdateServer/src/ESP8266HTTPUpdateServer.cpp (hunk 1 above)
Comment (caverna) [reply libraries/ESP8266HTTPUpdateServer/src/ESP8266HTTPUpdateServer.cpp]: Hi @PA4WD, do you need help with that?
This PR will be a great improvement to this feature!

**Comment by PA4WD** [comment]
updated my repro,
and changed as suggested.

**Comment by d-a-v** [comment]
Thanks, closed by #3732

**Pull request closed without merging**